src/index.ts

```diff
@@ -82,7 +82,7 @@ export type PlatformApi<
      *   after 7 seconds and the default number of tries is 20. On iOS, one try times out after 10 seconds and the
      *   default number of tries is 20.
      */
```

Comment:
We do want to document all options that are exposed. Especially since this could also be useful to others.